docs/src/content/hardhat-runner/docs/guides/hardhat-console.md

```diff
@@ -41,33 +43,35 @@ And the initialized `ethers` object if you're using the `hardhat-ethers` plugin:
 >
 ```
 
-And the `artifacts` object if you're using the `hardhat-truffle5` plugin, and so on.
+Anything that has been injected into the [Hardhat Runtime Environment] will be magically available in the global scope.
 
-Anything that has been injected into the [Hardhat Runtime Environment] will be magically available in the global scope. Alternatively, if you're the more explicit kind of developer, you can instead require the HRE explicitly and get autocomplete:
+Alternatively, if you're the more explicit kind of developer, you can instead require the HRE explicitly:
 
 ```
 > const hardhat = require("hardhat")
```

Comment:
Can we change this to use `hre` instead, for consistency with the rest of the docs?

Reply:
yeah, makes sense